docs/programs/token-metadata/changelog/v1.3.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "v1.3"
+sidebar_position: -3
+---
+
+# Version 1.3
+
+## New Features
+
+- Sized Collections
+- Burning NFTs
+- Set Token Standard
+
+## New Instructions
+
+- `CreateMetadataAccountV3`
+  - Same as `CreateMetadataAccount2`, but supports `CollectionDetails`
+- `VerifySizeCollectionItem`
+  - Allows a sized collection item `verified` flag to become `true` on an NFT to represent a Certified Collection. Essentially this officially adds to a Collection.
+- `UnVerifySizedCollectionItem`
+  - Allows a sized collection `verified` flag to become `false` on an NFT to represent a Certified Collection. Essentially this officially removes from a Collection.
+- `SetAndVerifySizedCollectionItem`
+  - Allows both setting and verifying, in the same instruction, the collection item for a sized collection.
+- `SetCollectionSize`
+  - Allows an unsized collecetion to set its size once to migrate to the new sized collection standard.
+- `SetTokenStandard`
+  - Allows a token without a token standard to have it set. The program automatically detects and sets the correct token standard type.
+- `BurnNft`
+  - Allows the owner of a NFT to burn it and reclaim funds locked up in token, metadata and edition accounts.
+
+
+## Deprecations
+
+- The following instruction now shows a deprecation warning when executed
+  - `CreateMetadataAccountV2` - _please use `CreateMetadataAccountV3` instead_
+
+## Removals
+
+Removal of previously deprecated instructions
+
+- 
```

Comment:
There's an empty bullet point here. Were you planning on listing the deprecated instructions that have been removed as of v1.3?

Reply:
Yeah, exactly, good catch! I'll fix that.